server/graphql-schema.js

```diff
@@ -1039,14 +980,15 @@ const graphqlSchema = gql`
         testPlanVersion(id: ID): TestPlanVersion
         """
         Load multiple TestPlanReports, with the optional ability to filter by
-        status, atId and testPlanVersionId.
+        TestPlanVersionPhase, atId, testPlanVersionId and if the report is approved.
         See TestPlanReport type for more information.
         """
         testPlanReports(
-            statuses: [TestPlanReportStatus]
+            phases: [TestPlanVersionPhase]
```

Comment:
Since the phase is actually on the testPlanVersion calling it just "phases" seems like it might mislead someone looking at it. It's more wordy, but I feel like "testPlanVersionPhases" communicates that the resolver is going to examine the associated testPlanVersion to filter down the list.

We discussed this, but I feel like so much progress has been made in the UI and in the CG that we can potentially make some big improvements to the overall ergonomics of the GraphQL API now. That would obviously not apply to this PR though.

Reply:
Agreed with using that terminology, it's quite helpful. I've updated `server/resolvers/testPlanReportsResolver` and the schema in https://github.com/w3c/aria-at-app/pull/715/commits/b92d12dfebd00954c11d29638dafae6d889f0468 to account for this